README.md

```diff
@@ -3,13 +3,23 @@
 Every player in the competition creates his version of of the ai-snake class.
 The players need to implement the 'change_direction' function so their snake will be 'smart' enough to survive and win.
 
+Some information about the game:
+
+Game settings: 
+1. We now need to specify the number of cells we want in the board. No need to mess with height, width or cell size.
+
+2. You can press + or - keys to adjust the delay between frames.
+
+
+Logic:
+1. When a snake hits itself is cuts itself in the hitting point.
+
+Fruits:
```

Comment:
Very nice fruits!
Add the normal strawberry, bomb and drangonfruit here also.
As for the king, a really nice implementation, looks great!
Maybe it could be a super good one - maybe give all fruit benefits together while it's king (the snake will be shielded, has a knife, and other things if we would add some).
Maybe also we would like to add a speed up \ speed down fruits as well.